docs/app/components/navigation-bar-search/navigation-bar-search.component.ts

```diff
@@ -34,46 +35,35 @@ export class NavigationBarSearchComponent {
     private data: ISearchResult[];
     private history: ISearchResult[] = [];
 
-    constructor(private router: Router, private navigation: NavigationService, private versionService: VersionService) {
-
-        // get version
-        this.versionService.version.subscribe((value: Version) => {
-            this.data = [];
-            this.data = this.data.concat(this.getSearchResults(componentsData));
-            this.data = this.data.concat(this.getSearchResults(cssData));
-            this.data = this.data.concat(this.getSearchResults(chartsData));
-        });
+    constructor(private router: Router, private navigation: NavigationService, private versionService: VersionService, private _appConfig: AppConfiguration) {
```

Comment:
Should private variables be prefixed with an underscore. i.e. `private _router: Router` ?

Reply:
Yes. This is what I used. `private _appConfig: AppConfiguration`